Geometry/MTDSimData/data/v2/mtdProdCuts.xml

```diff
@@ -3,23 +3,23 @@
 <DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
 
   <SpecParSection label="regions.xml" eval="true">
-    <SpecPar name="fasttime">                                                  
-      <PartSelector path="//BarrelTimingLayer"/>                          
-      <PartSelector path="//EndcapTimingLayer"/>                          
-      <Parameter name="CMSCutsRegion" value="FastTimerRegion" eval="false"/>   
-      <Parameter name="ProdCutsForElectrons" value="0.1*mm"/>                  
-      <Parameter name="ProdCutsForPositrons" value="0.1*mm"/>                  
-      <Parameter name="ProdCutsForGamma" value="0.1*mm"/>                      
-    </SpecPar>                                                                                  
+    <SpecPar name="fasttime">
+      <PartSelector path="//BarrelTimingLayer"/>
+      <PartSelector path="//EndcapTimingLayer"/>
+      <Parameter name="CMSCutsRegion" value="FastTimerRegion" eval="false"/>
+      <Parameter name="ProdCutsForElectrons" value="0.1*mm"/>
+      <Parameter name="ProdCutsForPositrons" value="0.1*mm"/>
+      <Parameter name="ProdCutsForGamma" value="0.1*mm"/>
+    </SpecPar>
     <SpecPar name="fasttime-sens">
       <PartSelector path="//BModule1Layer1Timingactive"/>
       <PartSelector path="//BModule17Layer1Timingactive"/>
       <PartSelector path="//BModule33Layer1Timingactive"/>
-      <PartSelector path="//LGAD_ActiveThickness"/>
-      <Parameter name="CMSCutsRegion" value="FastTimerRegion" eval="false"/>   
-      <Parameter name="ProdCutsForElectrons" value="0.1*mm"/>                  
-      <Parameter name="ProdCutsForPositrons" value="0.1*mm"/>                  
-      <Parameter name="ProdCutsForGamma" value="0.1*mm"/>                      
+      <PartSelector path="//LGAD_TimingActive"/>
+      <Parameter name="CMSCutsRegion" value="FastTimerRegion" eval="false"/>
+      <Parameter name="ProdCutsForElectrons" value="0.1*mm"/>
+      <Parameter name="ProdCutsForPositrons" value="0.1*mm"/>
+      <Parameter name="ProdCutsForGamma" value="0.1*mm"/>
```

Comment:
what about adding a production cut on Proton equal to the one on Electron: see issue https://github.com/cms-sw/cmssw/issues/27343?

Reply:
@ianna thanks for reminding me about this issue, I will update the production cuts in a next round, where I also update the name of some volumes to make the RECO geometry management easier